app/models/manageiq/providers/ansible_tower/configuration_manager/credential.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::Credential < Authentication
+end
```

Comment:
@jameswnl 
Do we need a factory for this under spec/factories
https://github.com/ManageIQ/manageiq/blob/master/spec/factories/authentication.rb#L50